src/Sarif.Viewer.VisualStudio/ErrorList/SarifTableDataSource.cs

```diff
@@ -84,42 +49,33 @@ public static SarifTableDataSource Instance
         }
 
         #region ITableDataSource members
-        public string SourceTypeIdentifier
-        {
-            get { return StandardTableDataSources.ErrorTableDataSource; }
-        }
 
-        public string Identifier
-        {
-            get { return Guids.GuidVSPackageString; }
-        }
+        public override string Identifier => Guids.GuidVSPackageString;
 
-        public string DisplayName
-        {
-            get { return Resources.ErrorListTableDataSourceDisplayName; }
-        }
+        public override string DisplayName => Resources.ErrorListTableDataSourceDisplayName;
 
-        public IDisposable Subscribe(ITableDataSink sink)
+        public override IDisposable Subscribe(ITableDataSink sink)
```

Comment:
>Subscribe [](start = 36, length = 9)

I can infer some information on Subscribe chattiness, etc., requested in my previous comment from this implementation. :)

Reply:
Not sure what you mean. What about the implementation do you notice, and what do you infer from it?

---
In reply to: [538760758](https://github.com/microsoft/sarif-visualstudio-extension/pull/313#discussion_r538760758) [](ancestors = 538760758)